### DISCUSSION TITLE
FluxStore Manager - add/edit Store information and logo  [done] ✅

### DISCUSSION THREAD

**Discussion by hopagy** [inspireui/support#5098] [Working on Features]
Hi,

I wish you add a page in FluxStore Manager for add/edit Store information, logo, header image, location map, etc...

Best regards,

**Comment by ImpactDesign91** [comment]
I think's it's a good idea ! And important feature ! Thank's to add this feature !

**Comment by philipvu30** [comment]
Hi,

This feature is implemented and will be available in the release 2.1.
Please note that you need to update the MStore Plugin to version 3.3.6.

Thank you.
